src/app/questions/questionhelper.ts

```diff
@@ -0,0 +1,15 @@
+import { Answer, Option, Question } from './questions.model';
+
+export class QuestionHelper {
+  static calculateMaxPoints(question: Question) {
```

Comment:
Why not make calculateMaxPoints an exported function? That way you can lose the class and import the functions directly, as the class serves no function here imho.